docs/configuring-metabase/custom-maps.md

```diff
@@ -19,27 +19,40 @@ To find the map settings:
+
+Currently, Metabase uses a single tile server per instance. You can't specify different tiles for different maps.
 
 ## Custom region maps
 
-If you need a map that focuses on a specific region, you may want to upload a custom GeoJSON map.
+Metabase comes with two built-in [region maps](../questions/visualizations/map.md#region-maps): world map with countries and United States map with states. If you need a map of other regions - like EU countries or NYC neighborhoods - you can upload a GeoJSON file containing region information.
+
+If you need a GeoJSON file for specific custom regions, you can often find it just by searching online for "[Your region] + geojson", like "NYC neighborhoods GeoJSON", because community members and government organization have already developed map files for many common regions. You can also create your own GeoJSON with a tool like [MapShaper](https://mapshaper.org/) or [GeoJSON.io](http://geojson.io/).
 
 Your GeoJSON file should:
 
 - Be less than 5 MB in size.
-- Use geographic coordinates (latitude and longitude). Metabase doesn't support projected coordinates, so you'll first need to convert projected coordinates to geographic coordinates.
+- Contain polygon features defining regions (not just points or coordinates)
+- Use geographic coordinates (latitude and longitude) to define region polygons. Metabase doesn't support projected coordinates, so you'll first need to convert projected coordinates to geographic coordinates.
+- Accessible by a public URL
+
+To add a custom map:
 
-To upload a GeoJSON file to add a custom region map, click the `Add a map` button. Enter:
+1. Go to **Admin Settings > Maps > Custom Maps**;
+2. Click "Add a Map";
+3. Enter a name for your map;
+4. Provide the URL to your GeoJSON file;
+5. Specify the JSON properties that should serve as region identifier and region name.
 
-- The name of the map
-- The URL of the GeoJSON file
-- The JSON property that Metabase should use as your region's identifier (a distinct way of identifying this specific region map)
-- The JSON property that Metabase should use as the display name (how the name of the region map appears in the user interface)
+![Uploading a custom GeoJSON](./images/custom-geojson.png)
 
-If you lack a GeoJSON file, you can create one with a tool like [MapShaper](https://mapshaper.org/) or [GeoJSON.io](http://geojson.io/).
+- **Name of the map** will be displayed in the region selector for [custom region maps](../questions/visualizations/map.md#custom-regions)
+- **Region's identifier** is a GeoJSON field that identifies your region. The values in this field should match how the regions are referenced in your data. It does not need to match the display name.
+- **Region's display name** is a GeoJSON field that contains how your region name will be displayed on a map. If can be different from region's identifier.
```

Comment:
```suggestion
- **Region's display name** is a GeoJSON field that specifies how your region name will be displayed on a map. It can be different from region's identifier.
```